homeassistant/helpers/device_registry.py

```diff
@@ -503,14 +503,16 @@ async def _async_migrate_func(
                 # Introduced in 2024.7
                 for device in old_data["devices"]:
                     device["primary_config_entry"] = None
-            if old_minor_version < 7:
+            if old_minor_version < 8:
```

Comment:
Please add a comment explaining why we skip version 7

Reply:
added